packages/mds-geography/api.ts

```diff
@@ -48,8 +57,8 @@ function api(app: express.Express): express.Express {
         ) {
           params.get_published = true
         }
-        const metadata = await db.readBulkGeographyMetadata(params)
-        return res.status(200).send(metadata)
+        const geography_metadata = await db.readBulkGeographyMetadata(params)
+        return res.status(200).send({ version: res.locals.version, geography_metadata })
```

Comment:
Would be nice to pull out `version` with a destructure like `const { version } = res.locals` so you could just say `return res.status(200).send({ version, geography_metadata })`

Reply:
(This is applicable to basically all the responses)

Reply:
Will do in future versioning PRs!